### PR TITLE
Adding Workshop Box to the manual

### PR DESCRIPTION
**Pull request by patjouk**
Added info on the Workshop Box.

### CODE REVIEW

File: faq/README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ If you can't do that, invite your coaches to meet their group online and send th
 
 ### Swag
 
-**Q: Do you provide swag for events?**
+**Q: Do you provide swag (stickers, nametags, etc) for events?**
 
-A: We don't for the moment. Don't hesitate to contact other organizers near your location on Slack or Google group: some may have extra from a previous event and may be happy to send them to you!
+A: Yes, but you have to meet those requirements: you need to contact us at least 6 weeks before your event and attendees registration must be open. To know more about this, check the [workshop box page](https://djangogirls.org/workshop-box) on Django Girls website. You will need to be logged in admin to be able to see this page.
```
Comment (phalt) [inline faq/README.md]: `meet these requirements`

File: logistics/README.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ There are a few small, cheap practical items that are really useful to have:
 
 ## Goodies
 
-This is only if you have the budget. 
+This is only if you have the budget. We can send you a [Workshop Box](https://djangogirls.org/workshop-box) for 50$ or less if you meet those requirements: you need to contact us at least 6 weeks before your event and attendees registration must be open. Got to the [Workshop Box page](https://djangogirls.org/workshop-box) to know more about it.
```
Comment (phalt) [inline logistics/README.md]: `these requirements` here too :)

**Comment by phalt** [comment]
👍 other than minor comments! 🍰

**Comment by patjouk** [comment]
Thanks @phalt :sparkles:

**Pull request merged**